odybcl2fastq/run.py

```diff
@@ -312,10 +312,9 @@ def copy_source_to_output(src_root, dest_root, sample_sheet, instrument):
     # final
     src_root += '/'
     dest_root += '/'
-    if instrument == 'hiseq':
-        run_params_file = 'runParameters.xml'
-    else:  # nextseq and novaseq
-        run_params_file = 'RunParameters.xml'
+    run_params_file = 'RunParameters.xml'
+    if not os.path.exists(src_root + run_params_file):
+        util.copy(src + 'runParameters.xml', src + run_params_file)
```

Comment:
Looks like `src` isn't defined; should it be `src_root`?

Reply:
yes, nice catch.  The run I tested had the RunParameters file already so didn't find this error. Thanks!